no_of_points: 8


point_0:
  name: center
  pose: [0.0,0.0]
  orientation: [0.0,1.0]
  adj_point: [0,1,4]
  connected_points: 1,4

point_1:
  name: point1
  pose: [10.3601999283,-0.0924104377627]
  orientation: [0.0,1.0]
  adj_point: [1,2,0]
  connected_points: 0,2,5

point_2:
  name: point2
  pose: [10.5281925201,13.4487886429]
  orientation: [1.0,0.0]
  adj_point: [2,3,1]
  connected_points: 1,3

point_3:
  name: point3
  pose: [-9.1704788208,13.1101837158]
  orientation: [0.0,1.0]
  adj_point: [3,4,2]
  connected_points: 4,2

point_4:
  name: point4
  pose: [-9.1833562851,0.02333021909]
  orientation: [1.0,0.0]
  adj_point: [4,3,0]
  connected_points: 0,3

point_5:
  name: point5
  pose: [10.9937171936,-4.6669216156]
  orientation: [-0.7,0.7]
  adj_point: [5,7,6,1]
  connected_points: 1,6,7

point_6:
  name: point6
  pose: [5.39780712128,-4.91615343094]
  orientation: [1.0,0.0]
  adj_point: [6,5,5]
  connected_points: 5

point_7:
  name: point7
  pose: [11.0155220032,-7.65437650681]
  orientation: [-0.7,0.7]
  adj_point: [7,5,5]
  connected_points: 5

